library-app/src/components/Layout/Modal.module.css

```diff
@@ -0,0 +1,24 @@
+.overlay {
+  position: fixed;
+  top: 0;
```

Comment:
Top, right, left and bottom all 0?

Reply:
Would it be cleaner to use width & height?

Reply:
it s the way i can cover entire screen